### PR TITLE
FEATURE: Player UI

### PR DESCRIPTION
**Pull request by jmysliv**
* Added web player UI
* Added mobile player UI with basic animation

TODO (In separate PRs, I don't want this pr to get too big.):
* Player smooth animation (Spotify-like)
* Player logic

Screenshot 2022-08-29 at 13 26 39

https://user-images.githubusercontent.com/48885911/187192254-47cdf65f-88d3-4b61-a00e-8d2fccf4b0ae.mp4

### CODE REVIEW

**Review by alejomendoza — APPROVED** [review]

File: packages/app/features/player/components/likesLIst.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import { User } from "app/api/graphql";
```
Comment (co012) [inline packages/app/features/player/components/likesLIst.tsx:1]: Fix typo in file name

File: packages/app/features/player/components/likesLIst.tsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+};
+
+export function LikesList({ likers }: Props) {
+  const renderItem = ({ item }: { item: User }) => {
+    return (
+      <View className="mr-2" key={item.id}>
+        <UserAvatar user={item} />
+      </View>
+    );
+  };
```
Comment (co012) [inline packages/app/features/player/components/likesLIst.tsx:18]: (Optional) I would extract it outsize LikesList component

File: packages/app/features/player/fullScreenPlayer.tsx [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+const { height } = Dimensions.get("window");
+
+type Props = {
+  onTogglePress: () => void;
+};
+
+export function FullScreenPlayer({ onTogglePress }: Props) {
+  return (
+    <SafeAreaView
+      className={`flex flex-column px-4 bg-blue-dark items-center absolute w-full h-${
```
Comment (co012) [inline packages/app/features/player/fullScreenPlayer.tsx:20]:
```suggestion
      className={`flex flex-col px-4 bg-blue-dark items-center absolute w-full h-${
```

You can drop flex-col, flex direction defaults to column

File: packages/app/ui/userAvatar.tsx [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+  let initials = "";
+  if (user.displayName) {
+    initials = compose(
+      join(""),
+      take(2),
+      map((part) => part[0]),
+      split(" ")
+    )(user.displayName);
+  }
```
Comment (co012) [inline packages/app/ui/userAvatar.tsx:27]: add toUpperCase

File: packages/app/features/player/playerBar.tsx [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+          >
+            Artist
+          </Text>
+        </View>
+      </View>
+      <View className="items-center justify-evenly flex-column w-100">
+        <PlayerButtonsRow />
+        <PlayerSlider />
+      </View>
+      <View className="w-50" />
```
Comment (co012) [inline packages/app/features/player/playerBar.tsx:31]: Is this some kind of padding?

**Review by co012 — APPROVED** [review]
LGTM

File: packages/app/features/player/playerBar.tsx (hunk 5 above)
Comment (jmysliv) [reply packages/app/features/player/playerBar.tsx:31]: I've added it so the slider could be aligned in the center of the row (I've used `justify-between`)

File: packages/app/features/player/components/likesLIst.tsx (hunk 2 above)
Comment (jmysliv) [reply packages/app/features/player/components/likesLIst.tsx:18]: As the `LikesList` component is quite small, I think we can leave it there.

**Pull request merged**